### DISCUSSION TITLE
有没有办法帮助识别广告花体字/艺术字字

### DISCUSSION THREAD

**Discussion by magicliang** [PaddlePaddle/PaddleOCR#13843] [Ideas & Features]
![ocr2](https://github.com/user-attachments/assets/7869ce43-e48d-4621-b598-f3a105b25f8e)
![ocr1](https://github.com/user-attachments/assets/b66bedee-3426-456b-96ac-b883445c6aa7)
![ocr3](https://github.com/user-attachments/assets/f46634c0-b826-487a-8697-192f24b87a4f)
我们有很特别的场景需要把这些图里的文字完整地识别出来，但试过 PP-OCRv4 PP-ChatOCRv2-common 好像都不能完整地识别这些花体字/艺术字。

**Comment by GreatV** [comment]
要在自己的场景下重新训练一下

参考 https://paddlepaddle.github.io/PaddleOCR/ppocr/model_train/training.html#pp-ocr

**Comment by UserWangZz** [comment]
艺术字检测应该效果不错，但是识别目前仍然是一个很hard的问题、
